app/src/main/res/values/styles.xml

```diff
@@ -459,4 +459,9 @@
     <item name="android:gravity">center_horizontal</item>
   </style>
 
+  <!-- ShapeableImageView -->
```

Comment:
Adding this comment as a placeholder to continue the discussion from https://github.com/oppia/oppia-android/pull/4139#discussion_r796232926.

Reply:
I took this document as reference for this property.
[](https://github.com/oppia/oppia-android/pull/4139#discussion_r796442630)https://howtodoandroid.com/shapeableimageview-material-components-android/

Reply:
Thanks. I suggest adding a comment pointing to that page for context, then, as it'll help others understand why it's correct if they come across this in the future (unfortunately, the proper name makes this a bit difficult to understand hence why the comment will have contextualize).

Reply:
Done